addons/source-python/packages/source-python/listeners/tick.py

```diff
@@ -109,19 +109,17 @@ def add(self, delay):
 class Delay(WeakAutoUnload):
     """Execute a callback after a given delay."""
 
-    def __init__(
-        self, delay, callback, cancel_on_map_end=False, *args, **kwargs
-    ):
+    def __init__(self, delay, callback, cancel_on_map_end=False, args=(), kwargs={}):
```

Comment:
You do realize that threading itself uses this better practice, right?

https://github.com/python/cpython/blob/master/Lib/threading.py#L758

https://github.com/python/cpython/blob/master/Lib/threading.py#L781#L782

https://github.com/python/cpython/blob/master/Lib/threading.py#L786

https://github.com/python/cpython/commit/6fe467c15d15a92fdc843af688363a96060a3554

Code is more often read than written.  Someone could very easily mistake your implementation and it would throw them off.